packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -1645,14 +1659,18 @@ export async function createManifest(
   hasFrontend: boolean,
   hasBot: boolean,
   hasMessageExtension: boolean,
-  isSPFx: boolean
+  isSPFx: boolean,
+  hasAad = true
 ): Promise<TeamsAppManifest | undefined> {
-  if (!hasBot && !hasMessageExtension && !hasFrontend) {
+  if (!hasBot && !hasMessageExtension && !hasFrontend && !hasAad) {
```

Comment:
will there be cases that we have no bot/tab/message extension, but have AAD ?

Reply:
I think we do not have such scenario now. Here I try to make it clear since it won't have any influence on current logic, but may benefit if we have such scenario in the future.